scripts/scil_screenshot_bundle.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import argparse
+import os
+
+from dipy.io.stateful_tractogram import Space, StatefulTractogram
+from dipy.io.streamline import load_tractogram
+from dipy.tracking.streamline import transform_streamlines
+from fury import actor
+import matplotlib.pyplot as plt
+import nibabel as nib
+import numpy as np
+from scipy.ndimage import map_coordinates
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+from scilpy.utils.image import register_image
+from scilpy.viz.screenshot import display_slices
+
+DESCRIPTION = """
+   Register bundle to a template for screenshots using a reference.
+   The template are in /mnt/braindata/Other/simple_template_viz/
+   Axial, coronal and sagittal slices are captured.
+   Sagittal can be capture from the left (default) or the right.
+   """
+
+
+def _build_args_parser():
+    p = argparse.ArgumentParser(
+        description=DESCRIPTION,
+        formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_bundle',
+                   help='Path of the input bundle')
+    p.add_argument('in_anat',
+                   help='Path of the reference file (.nii or nii.gz)')
+    p.add_argument('target_template',
+                   help='Path to the target MNI152template for registration, '
+                        'Any choice of modality works')
+
+    sub_color = p.add_mutually_exclusive_group()
+    sub_color.add_argument('--local_coloring', action='store_true',
+                           help='Color streamlines local segments orientation')
+    sub_color.add_argument('--uniform_coloring', nargs=3, metavar=('R', 'G', 'B'),
+                           type=int,
+                           help='Color streamlines with uniform coloring')
+    sub_color.add_argument('--reference_coloring',
+                           metavar='COLORBAR',
+                           help='Color streamlines with reference coloring (0-255)')
+
+    p.add_argument('--right', action='store_true',
+                   help='Take screenshot from the right instead of the left \n'
+                        'for the sagittal plane')
+    p.add_argument('--anat_opacity', type=float, default=0.3,
+                   help='Set the opacity for the anatomy, use 0 for complete '
+                   'transparency, 1 for opaque')
+    p.add_argument('--output_suffix',
+                   help='Add a suffix to the output, else the axis name is used')
+    p.add_argument('--output_dir', default='',
+                   help='Put all images in a specific directory, will overwrite')
```

Comment:
The overwrite param makes it sure that it will not overwrite, update help message.